### PR TITLE
Cr 1795 cc bf postcodes

### PR DESCRIPTION
**Pull request by PeterBochel**
Changes:

- Remove BFPO from list of valid postcodes. This will stop them getting to AI where there are rejected/fail.
- Updated swagger to show latest post query regex.

### CODE REVIEW

File: swagger-future.yml [hunk 1]
```diff
@@ -1269,7 +1270,7 @@ components:
 
     POSTCODE_CONST:
       type: string
-      pattern: 'GIR[ ]?0AA|((AB|AL|B|BA|BB|BD|BH|BL|BN|BR|BS|BT|BX|CA|CB|CF|CH|CM|CO|CR|CT|CV|CW|DA|DD|DE|DG|DH|DL|DN|DT|DY|E|EC|EH|EN|EX|FK|FY|G|GL|GY|GU|HA|HD|HG|HP|HR|HS|HU|HX|IG|IM|IP|IV|JE|KA|KT|KW|KY|L|LA|LD|LE|LL|LN|LS|LU|M|ME|MK|ML|N|NE|NG|NN|NP|NR|NW|OL|OX|PA|PE|PH|PL|PO|PR|RG|RH|RM|S|SA|SE|SG|SK|SL|SM|SN|SO|SP|SR|SS|ST|SW|SY|TA|TD|TF|TN|TQ|TR|TS|TW|UB|W|WA|WC|WD|WF|WN|WR|WS|WV|YO|ZE)(\\d[\\dA-Z]?[ ]?\\d[ABD-HJLN-UW-Z]{2}))|BFPO[ ]?\\d{1,4}'
+      pattern: 'GIR[ ]?0AA|(AB|AL|B|BA|BB|BD|BH|BL|BN|BR|BS|BT|BX|CA|CB|CF|CH|CM|CO|CR|CT|CV|CW|DA|DD|DE|DG|DH|DL|DN|DT|DY|E|EC|EH|EN|EX|FK|FY|G|GL|GY|GU|HA|HD|HG|HP|HR|HS|HU|HX|IG|IM|IP|IV|JE|KA|KT|KW|KY|L|LA|LD|LE|LL|LN|LS|LU|M|ME|MK|ML|N|NE|NG|NN|NP|NR|NW|OL|OX|PA|PE|PH|PL|PO|PR|RG|RH|RM|S|SA|SE|SG|SK|SL|SM|SN|SO|SP|SR|SS|ST|SW|SY|TA|TD|TF|TN|TQ|TR|TS|TW|UB|W|WA|WC|WD|WF|WN|WR|WS|WV|YO|ZE)(\\d[\\dA-Z]?[ ]?\\d[ABD-HJLN-UW-Z]{2}))'
```
Comment (rpcatling) [inline swagger-future.yml:1273]: think you have left one too many closing parenth at the end.

File: swagger-current.yml [hunk 2]
```diff
@@ -1,13 +1,14 @@
 openapi: 3.0.0
 info:
   title: ONS Contact Centre API
-  version: "5.10.17-oas3"
+  version: "5.11.18-oas3"
```
Comment (rpcatling) [inline swagger-current.yml:4]: interesting that the versions got switched from 5.10.xx to 5.11.xx  . Wonder if that. was intentional or by accident ? Might be worth double checking.

**Review by rpcatling — CHANGES_REQUESTED** [review]
Assuming that CCSvc will have unit tests to verify the regex when that is done. 

Minor discrepancy spotted.

File: swagger-future.yml (hunk 1 above)
Comment (PeterBochel) [reply swagger-future.yml:1273]: Removed extra parenthesis

File: swagger-current.yml (hunk 2 above)
Comment (PeterBochel) [reply swagger-current.yml:4]: I took this as being correct. Mentioned to Phil yesterday but he didn't seem interested

**Review by rpcatling — APPROVED** [review]

**Review by philwhiles — APPROVED** [review]

**Pull request merged**